routers/common/middleware.go

```diff
@@ -57,11 +57,12 @@ func Middlewares() []func(http.Handler) http.Handler {
 			// Why we need this? The Recovery() will try to render a beautiful
 			// error page for user, but the process can still panic again, and other
 			// middleware like session also may panic then we have to recover twice
-			// and send a simple error page that should not panic any more.
+			// and send a simple error page that should not panic anymore.
 			defer func() {
 				if err := recover(); err != nil {
-					combinedErr := fmt.Sprintf("PANIC: %v\n%s", err, string(log.Stack(2)))
-					log.Error("%v", combinedErr)
+					UpdateContextHandlerPanicError(req.Context(), err)
+					combinedErr := fmt.Sprintf("PANIC: %v\n%s", err, log.Stack(2))
+					log.Error("panic occurred and gets recovered, %v", combinedErr)
```

Comment:
Same as above, IMHO "panic occurred and gets recovered" adds nothing and makes it harder to see the panic.

```suggestion
					log.Error("%v", combinedErr)
```

Reply:
As above.